README.md

```diff
@@ -3,13 +3,25 @@
 [![Build Status: Ubuntu Bionic (Actions)](https://github.com/ros-industrial/universal_robot/workflows/CI%20-%20Ubuntu%20Bionic/badge.svg?branch=melodic-devel)](https://github.com/ros-industrial/universal_robot/actions?query=workflow%3A%22CI+-+Ubuntu+Bionic%22)
 [![Build Status: Ubuntu Focal (Actions)](https://github.com/ros-industrial/universal_robot/workflows/CI%20-%20Ubuntu%20Focal/badge.svg?branch=melodic-devel)](https://github.com/ros-industrial/universal_robot/actions?query=workflow%3A%22CI+-+Ubuntu+Focal%22)
 
-[![license - apache 2.0](https://img.shields.io/:license-Apache%202.0-yellowgreen.svg)](https://opensource.org/licenses/Apache-2.0)
+[![License - apache 2.0](https://img.shields.io/:license-Apache%202.0-yellowgreen.svg)](https://opensource.org/licenses/Apache-2.0)
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+[![License](https://img.shields.io/badge/License-Universal%20Robots%20A/S%E2%80%99%20Terms%20and%20Conditions%20for%20Use%20of%20Graphical%20Documentation-blue.svg)](https://www.universal-robots.com/legal/terms-and-conditions/terms_and_conditions_for_use_of_graphical_documentation.txt)
+
+[![support level: community](https://img.shields.io/badge/support%20level-community-lightgray.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
+
 [![support level: community](https://img.shields.io/badge/support%20level-community-lightgray.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
 
 [ROS-Industrial](https://wiki.ros.org/Industrial) Universal Robot meta-package. See the [ROS wiki](https://wiki.ros.org/universal_robots) page for compatibility information and other more information.
 
+__License__
+
+The [UR20 meshes](ur_description/meshes/ur20) constitutes “Graphical Documentation” the use of which is subject to and governed by our “[Terms and Conditions for use of Graphical Documentation](https://www.universal-robots.com/legal/terms-and-conditions/terms_and_conditions_for_use_of_graphical_documentation.txt)”.\
```

Comment:
```suggestion
Note: the mesh files used as part of the [UR20](ur_description/meshes/ur20) model in the `ur_description` package come with a non-OSI approved, non-open-source license.
They are considered "Graphical Documentation" by Universal Robots and their use is subject to and governed by the "[Terms and Conditions for use of Graphical Documentation](https://www.universal-robots.com/legal/terms-and-conditions/terms_and_conditions_for_use_of_graphical_documentation.txt)".

Users of these packages are advised to make sure their use complies with these additional terms and conditions.
Please read the license carefully before making use of the UR20 variant in the `ur_description` package.
```

Edit: I would actually suggest to make the link here *also* point to `ur_description/meshes/ur20/LICENSE.txt`, just as the badge (see earlier comment).

I don't know what the link would become though, so I don't have a suggestion to easily accept and apply I'm afraid.

Reply:
@urrsk as far as I remember this section is an outcome of UR's law department, so I guess I cannot say much to that.

Reply:
The original section contained style violations and typos. Additionally, we can't have it state "our [license]", as this is a repository on the `ros-industrial` organisation. It's not owned by UR, and we are not UR.

The rest of the changes merely draw attention to the fact that legally, users should *not* use any of the files (and perhaps not even download them) under the new license until after they've determined they can and will agree to its terms.

I'm sure lawyers will be able to find fault with just about anything, but I can't really see what's wrong here. It's definitely a non-OSI approved, non-OSS license. That's just a fact, and I feel important to call out.

Reply:
@gavanderhoorn wrote:
> The rest of the changes merely draw attention to the fact that legally, users should _not_ use any of the files (and perhaps not even download them) under the new license until after they've determined they can and will agree to its terms.

which is exactly what the new license also states/asks (from the preamble of `v1.01`):
> The T&Cs constitute a binding agreement between you and Universal Robots. By accessing, storing, copying, sharing, opening, or otherwise using or disposing of the Graphical Documentation, you acknowledge that you are bound by the T&Cs.

Reply:
@gavanderhoorn thanks for your feedback.
Agree, the wording is made for a repo UR's Github organisation.
I will come back with another suggestion.
And yes it is not a OSI approved license